drivers/javascript/cursor.coffee

```diff
@@ -255,6 +246,9 @@ class IterableResult
         return nextCb().nodeify(errCb)
     )
 
+    _each: @::each
+    _eachAsync: @::eachAsync
```

Comment:
These lines are really just for backwards-compatibility. We could remove them if we update the few references of `@_each` to `@each` in this class.